scripts/humkin2_herb.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+import logging 
+import rospy
+import herbpy
+
+
+if __name__ == "__main__":   
+    logger = logging.getLogger('test_skel_herb')
+    logger.setLevel(logging.INFO)
+    
+    rospy.init_node('humkin2') 
+    segway_sim = rospy.get_param("~seg_sim");    
+    
+    env, herb = herbpy.initialize(attach_viewer='InteractiveMarker', segway_sim=segway_sim)
+
+    herb.DetectHuman(env, orhuman=2, segway_sim=segway_sim)
+
+
+
```

Comment:
I now see this is in the main readme. However I think it should also be in the scrips folder